tokio-test/tests/io.rs

```diff
@@ -84,3 +87,67 @@ async fn mock_panics_write_data_left() {
     use tokio_test::io::Builder;
     Builder::new().write(b"write").build();
 }
+
+#[tokio::test]
+async fn wait() {
+    // 200ms should be enough to ensure no other factors are causing the delay
+    const FIRST_WAIT: Duration = Duration::from_millis(200);
```

Comment:
If we use Tokio's time mocking feature, then the test will run instantly without there being any chance of timing affecting the test. It also lets us use simpler durations.
```suggestion
#[tokio::test(start_paused = true)]
async fn wait() {
    const FIRST_WAIT: Duration = Duration::from_secs(1);
```

Reply:
f1edf88 makes use of the time mocking feature using tokio's `Duration` and `Instant`. Thanks for letting me know this is great I didn't know this existed.